src/orca/services/nextflowtower/models.py

```diff
@@ -321,6 +321,10 @@ class Workflow(BaseTowerModel):
         "state": "status",
     }
 
+    def __repr__(self) -> str:
+        """String representation of a workflow."""
+        return f"Workflow(run_name={self.run_name}, id={self.id}, state={self.state})"
+
```

Comment:
Thoughts on adding a `__str__` representation too that is just the id?

Reply:
Dataclasses already have default implementations for `__repr()__` and `__str__()`. I'm just overriding this one because the full output is too long for logging, which I want both the human-friendly run name and computer-friendly ID to appear. I'm aware of the `field(..., repr=False)` parameter, but I don't want to set that for all but a handful of attributes.